CONTRIBUTING.md

```diff
@@ -17,16 +17,19 @@ again.
 
 ## Building Jib
 
-Jib comes as 3 components:
+Jib comes as 3 public components:
 
   - `jib-core`: a library
-  - `jib-plugins-common`: a library with helpers for plugin builders
   - `jib-maven-plugin`: a Maven plugin that uses `jib-core` and `jib-plugins-common`
   - `jib-gradle-plugin`: a Gradle plugin that uses `jib-core` and `jib-plugins-common`
 
-To build, use the provided `build.sh` which builds and tests each of the
-components into your local `~/.m2/repository`.  Note that this script does
-not run integration tests.
+And 1 internal component:
+
+  - `jib-plugins-common`: a library with helpers for plugin builders
+
+The project is configured as a single gradle build, to build the whole
+project run `./gradlew build`, to install into the local maven repository
+run `./gradlew install`.
```

Comment:
This might be punctuated a bit more clearly: "The project is configured as a single gradle build. Run `./gradlew build` to build the whole project, or run `./gradlew install` to install into the local maven repository."